.github/workflows/update_sdk_methods.py

```diff
@@ -1543,6 +1543,9 @@ def write_markdown(type, names, methods):
                             file_contents = regex.sub(r'\{\{\%.*\%\}\}.*\{\{\% \/[a-b].* \%\}\}', '', file_contents, flags=regex.DOTALL)
                             search_result = file_contents.split('.\n', 1)[0].strip().replace("\n", " ")
 
+                            ## If the proto description contains any MD links, strip them out:
```

Comment:
wait so the proto link to the python docs?? Maybe we should remove that instead

Reply:
let's look into it later and merge this for now. I think it's confusing to point to python docs when someone might be looking at another language

Reply:
Oh! I see what you mean now. Forest for trees moment for me. You're absolutely right, I'll convert any SDK-specific links in proto descriptions to point to docs content instead in a follow-on. Thank you!